validation/validation_test.go

```diff
@@ -63,15 +63,15 @@ func testUniversal(t *testing.T, chain *ChainConfig) {
 // i.e. not to a Standard Candidate Chain.
 func testStandardCandidate(t *testing.T, chain *ChainConfig) {
 	t.Run("Standard Config Params", func(t *testing.T) {
-		testDataAvailability(t, chain)
-		testResourceConfig(t, chain)
-		testL2OOParams(t, chain)
-		testGasLimit(t, chain)
-		testGasPriceOracleParams(t, chain)
+		t.Run("Data Availability", func(t *testing.T) { testDataAvailability(t, chain) })
+		t.Run("Resource Config", func(t *testing.T) { testResourceConfig(t, chain) })
+		t.Run("L2OO Params", func(t *testing.T) { testL2OOParams(t, chain) })
+		t.Run("Gas Limit", func(t *testing.T) { testGasLimit(t, chain) })
+		t.Run("GPO Params", func(t *testing.T) { testGasPriceOracleParams(t, chain) })
```

Comment:
What is the advantage of wrapping these test functions with the `t.Run()` method?

Reply:
This is the only way we can exclude a chain from some of these tests but not others. With the `.Run()` syntax, the test is executed as a _named_ sub test, which we can target in the `exclusions_test.go` file.